setup.py

```diff
@@ -172,16 +169,19 @@ def run(self):
         sdist.run(self)
 
 
+install_requires = ["numpy", "pillow"]
+
 extras_require = {
+    "linting": ["black", "flake8"],
+    "dev": ["invoke", "pytest", "pytest-cov"],
```

Comment:
Why split out linting? Anyone who does some dev work will also want to run `invoke format` (which requires black), and probably lint as well.

Reply:
because black doesn't build on `pypy` and this was the fastest way to separate the lining dependencies and the testing dependencies.

I think a nice change (if you agree) would be to rename `dev` to `test` and add a new `dev` line, which combines `linting` and `testing`. This way the workflows can install `test`/`lint` respectively while developers can install `dev`

Reply:
Cool, yeah let's do that.